content/guides/process.md

```diff
@@ -1,23 +1,80 @@
+
+ECS Containers are tagged by `#task_name`, `#task_version`, and `#ecs_cluster`
+
+Kubernetes Containers are tagged by `#pod`, `#pod_ip`, `#service`, `#namespace`, `#cluster-name`, `#replica_set`, `#daemon_set`, and `#deployment`.
+
+**Filtering and Pivoting**
+
+Making sense of hundreds of thousands or millions of processes and containers can seem overwhelming!  Using tagging, described in the previous section, makes navigation easy.
+
+In the below, we have filtered down to a Kubernetes cluster of 9 nodes.  RSS and CPU utilization on containers is reported compared to the hard limits set on the containers, when they exist.  Here, we see that the containers in this cluster are way overprovisioned, and that we could use tighter limits and bin packing to acheive better utilization of resources.
+
+{{< img src="process/overprovisioned.png" >}}
+
+Container environments are dynamic and can be hard to follow.  Here, we pivot by `#service` and `#host`, and to reduce system noise, filter to `#namespace:default`, and we can see what services are running where, and how saturated key metrics are.  
+
+{{< img src="process/hostxservice.png" >}}
 
-## Configure the Datadog Agent to collect Live Process data
+It would be easy to pivot by ECS `#task_name` and `#task_version` and understand changes to resource utilization between updates.
+
+{{< img src="process/tasksxversion.png" >}}
+
+Below, we have searched for ssh processes and pivoted by `#user` to understand who is logged into which hosts.
+
+{{< img src="process/sshusers.png" >}}
+
+Ok, so I guess that last one is less exciting after redaction!
+
+## Broad Inspection, Deep Inspection
+
+Everyone's workflow differs.  Initially the table is displayed at the finest grain, but with the group-by field, you should start your investigation where it's appropriate for you: Grouping by Availability Zone, Host, Cluster, Pod, or wherever.
+
+From there, you can dig down into finer grains, or inspect each group to see individual processes or containers.  In the below screenshot, you can see an investigation that started by indexing by pod and service, dug into one pod to see the containers, and then expanded a container to see the process tree inside.  In the container inspect tray, we also have some recent context for these metrics.
+
+{{< img src="process/containerinspect.png" >}}
+
+## Real-time monitoring
+
+While actively working with the Process and Containers page, metrics are collected at 2s resolution.  This is very important for highly volatile metrics such as CPU.
+
+In the background, for historical context, metrics are collected at 10s resolution.  To disable realtime mode, edit your `datadog.conf` file as described in the installation section below and append:
+
+    [process.config]
+    realtime_interval: 10
```

Comment:
this flag might change in the next version unless it's used right now. we can keep backwards compatibility if needed. (this would be the last time we'd change any flag names like this)